templates/less/pages/signup.less

```diff
@@ -0,0 +1,111 @@
+.signup-page {
+
+  .navbar {
+    background: transparent;
+  }
+
+  @white: /* not #F9F7FF...? */ white;
+  @dim: 6rem;
+  @border: 1.7px solid rgba(255,255,255,0.7);
+  @contentWidth: 47rem;
+
+  width: 100vw;
+  height: 100vh;
+  background: url(img/background.jpg);
+  background-size: 100% 100%;
+
+  h1, h2 {
+    color: @white;
+    text-shadow: none;
+    text-align: center;
+  }
+
+  .wordmark {
+    display: inline-block;
+    color: @white;
+  }
+
+  div[role=form] {
+    background: @white;
+    padding: 4rem;
+    margin: auto;
+    width: @contentWidth;
+    border-radius: 0.5rem;
+
+    label {
+      display: block;
+      input {
+        height: @dim;
+        font-weight: normal;
+        font-size: 2rem;
+      }
+      span {
+        font-weight: normal;
+      }
+    }
+
+    #feedback {
+      display: none;
+
+      & + span {
```

Comment:
`&` is the LESS code for "the current selector", so here it unpacks to `#feedback + span`, i.e. a span that follows the `<input id=feedback` element